tasks/util.py

```diff
@@ -248,33 +256,34 @@ def generate_tile_summary(session, table_id, column_id, tablename, colname):
         CREATE TEMPORARY TABLE raster_empty_{tablename_ns} AS
         SELECT ROW_NUMBER() OVER () AS id, rast FROM (
           WITH tilesize AS (SELECT
-            CASE WHEN ST_Area(the_geom) > 5000 THEN 250
-                 ELSE 50 END AS tilesize
-            FROM observatory.obs_table
-            WHERE id = '{table_id}'
+            CASE WHEN SUM(ST_Area({colname})) > 5000 THEN 250
```

Comment:
Need to check whether this causes performance degradation.

`the_geom` will drop very small isolated geoms right now.